server/base/src/main/java/org/apache/accumulo/server/conf/ServerConfigurationFactory.java

```diff
@@ -75,6 +79,12 @@ public ServerConfigurationFactory(ServerContext context, SiteConfiguration siteC
     this.siteConfig = siteConfig;
     this.systemConfig = memoize(() -> new SystemConfiguration(context,
         SystemPropKey.of(context.getInstanceID()), siteConfig));
+    tableParentConfigs =
+        Caffeine.newBuilder().expireAfterAccess(CACHE_EXPIRATION_HRS, TimeUnit.HOURS).build();
```

Comment:
I was wondering if adding a weigher would make sense.  Looking at the [caffine docs](https://github.com/ben-manes/caffeine/wiki/Eviction#size-based) the weight is only acquired when something is added to the map.  So it would not really handle the case of configuration growing in size (like a user adds a large prop after its in the cache).  So a weigher probably does not makes sense for this case because Configuration objects have a dynamic size.